internal/alerts/ci-dev-aks-mac-eus.json

```diff
@@ -10,9 +10,10 @@
             "apiVersion": "2023-03-01",
             "location": "eastus",
             "properties": {
-                "description": "rule group for cluster /subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourceGroups/ci-dev-aks-mac-eus-rg/providers/Microsoft.ContainerService/managedClusters/ci-dev-aks-mac-eus in MAC: /subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourceGroups/ci-dev-aks-mac-eus-rg/providers/microsoft.monitor/accounts/ci-dev-aks-eus-mac",
+                "description": "rule group for cluster ci-dev-aks-mac-eus in MAC: ci-dev-aks-eus-mac",
                 "scopes": [
```

Comment:
Hi Soham, i see you have hard coded cluster name in each query and have a file per cluster. But rather simplify this and just have 1 file with scopes listed out & commented (for cluster resouce ids, workspace resource ids & cluster names). or even have them in a text file. Those are the only 3 params that is different for each cluster, as the rules are all the same.

Reply:
Thanks Vishwa! I have updated the alert rules and created a file with the list of parameters and now we have a single file for alerts.